website/docs/docs/deploy/ci-jobs.md

```diff
@@ -63,12 +63,13 @@ If you're not using dbt Cloud’s native Git integration with [GitHub](/docs/cl
 
 
 1. Set up a CI job with the [Create Job](/dbt-cloud/api-v2#/operations/Create%20Job) API endpoint using `"job_type": ci` or from the [dbt Cloud UI](#set-up-ci-jobs).
-1. Call the [Trigger Job Run](/dbt-cloud/api-v2#/operations/Trigger%20Job%20Run) API endpoint to trigger the CI job. You must include these fields to the payload:
-   - Provide the pull request (PR) ID with one of these fields, even if you're using a different Git provider (like Bitbucket). This can make your code less human-readable but it will _not_ affect dbt functionality. 
+1. Call the [Trigger Job Run](/dbt-cloud/api-v2#/operations/Trigger%20Job%20Run) API endpoint to trigger the CI job. You must include both of these fields to the payload:
+   - Provide the pull request (PR) ID using one of these fields:
 
       - `github_pull_request_id`
       - `gitlab_merge_request_id`
-      - `azure_devops_pull_request_id` 
+      - `azure_devops_pull_request_id`
+      - `non_native_pull_request_id` (for example, BitBucket)
```

Comment:
i saw that https://github.com/dbt-labs/dbt-cloud/pull/8960 was closed so thought the change didn't get merged. but, i recently ran across this field when i was searching around the dbt-cloud repo for something else. happy to remove if it's not supposed to be here, just lemme know